pages/apim/3.x/user-guide/publisher/promote-api.adoc

```diff
@@ -0,0 +1,60 @@
+= Promote APIs
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_promote_apis.html
+:page-folder: apim/user-guide/publisher
+:page-layout: apim3x
+:page-liquid:
+
+== Overview
+
+The following sections describe how you can promote an API from an environment to another one.
+
+For a quick introduction to create an API in APIM, see the link:/apim/3.x/apim_publisherguide_create_apis.html[Create an API^].
+
+NOTE: This feature is available from APIM 3.10.0 and need your installation to be linked to Cockpit. For more information about Cockpit, see the link:/cockpit/1.x/cockpit_overview_introduction.html[Cockpit]
+
+== Context
+
+* Two installations
+** One linked to Demo and Production environments
+** Another one linked to Dev and QA environments
+** An API to promote
++
+image::apim/3.x/api-publisher-guide/promote-apis/graviteeio-promote-api-cockpit-graph.png[Map of installations in Cockpit]
+
+== Request promotion for an API
+
+Here, we will see how to promote our API from environment `DEV` to environment `QA`.
+Following actions are done with an API Publisher on `DEV` environment.
+
+. Go to the API to promote and click on image:icons/promote-button.png[role="icon"] *PROMOTE* button
++
+image::apim/3.x/api-publisher-guide/promote-apis/graviteeio-promote-api-promote-1.png[Promote button]
++
+. A window listing the environments available for the API promotion will pop up
++
+image::apim/3.x/api-publisher-guide/promote-apis/graviteeio-promote-api-promote-2.png[Target environments list]
++
+NOTE: If a promotion request already exists for your API on an environment, then you cannot do another request until the previous one is rejected or accepted.
++
+image::apim/3.x/api-publisher-guide/promote-apis/graviteeio-promote-api-promote-2-bis.png[Target environments list]
++
+. Choose the environment on which you want to promote the API, and click on *PROMOTE* button
+
+NOTE: When promoting an API, *members* and *groups* information are lost.
+
+NOTE: If a promotion has already been accepted for an API, you can request for a new promotion on the same target environment. When this promotion will be accepted, it will update the API.
```

Comment:
```suggestion
NOTE: If a promotion has already been accepted for an API, you can request a new promotion on the same target environment. Once this promotion is accepted, it will update the API.
```